google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudConfigurable.java

```diff
@@ -69,18 +96,14 @@ public JComponent createComponent() {
 
   @Override
   public boolean isModified() {
-    return cloudSdkPanel.isModified();
+    return false;
```

Comment:
won't this always return false anyway if theres nothing to configure from this screen now? Even if so, you may be right that its better to return an explicit false.

Reply:
i don't think there is an adapter as far as I know. I think returning an explicit false like you have is fine since it makes the contract clear.